### PR TITLE
fix(frontend): Update the link text to the GCP CAIP Notebook

### PR DESCRIPTION
**Pull request by jiyongjung0**
**Description of your changes:**
The link creates a Notebook with TF 2.3 instead of 2.1. I think that it might be better to just state that it is a notebook.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: frontend/src/pages/GettingStarted.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const PAGE_CONTENT_MD = ({
 
 ## Build your own pipeline with
 
-  * TensorFlow Extended (TFX) [SDK](https://www.tensorflow.org/tfx/guide) with end-to-end ML Pipeline Template ([Open TF 2.1 Notebook](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb))
+  * TensorFlow Extended (TFX) [SDK](https://www.tensorflow.org/tfx/guide) with end-to-end ML Pipeline Template ([Open a GCP Cloud AI Platform notebook](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb))
```
Comment (Bobgy) [inline frontend/src/pages/GettingStarted.tsx:53]: Nit: I heard that we should not use GCP in docs, always spell out as Google Cloud.

**Review by Bobgy — COMMENTED** [review]
The test failed because you need to update a test snapshot. Can you refer to frontend development guide in frontend/README.md?

**Comment by Bobgy** [comment]
Thank you for the fix!

File: frontend/src/pages/GettingStarted.tsx (hunk 1 above)
Comment (jiyongjung0) [reply frontend/src/pages/GettingStarted.tsx]: Updated to `Cloud AI Platform Notebook on Google Cloud` Thanks!

**Pull request merged**